.stylelintrc

```diff
@@ -0,0 +1,70 @@
+{
+  "extends": [
+    "stylelint-config-standard-scss",
+    "stylelint-config-idiomatic-order"
+  ],
+  "plugins": [
+    "stylelint-declaration-strict-value",
+    "@stylistic/stylelint-plugin"
+  ],
+  "overrides": [
+    {
+      "files": ["src/assets/scss/*.{css,scss}"],
+      "customSyntax": "postcss-scss",
+      "rules": {
+        "media-feature-range-notation": "prefix",
+        "color-function-notation": "legacy",
+        "shorthand-property-no-redundant-values": null,
+        "declaration-block-no-redundant-longhand-properties": [true, {
+          "ignoreShorthands": [
+            "inset",
+            "inset-block",
+            "inset-inline",
+            "mask",
+            "overscroll-behavior",
+            "padding-block",
+            "padding-inline",
+            "place-content",
+            "place-items",
+            "place-self",
+            "scroll-margin",
+            "border-block",
+            "border-block-end",
+            "border-block-start",
+            "border-inline",
+            "border-inline-end",
+            "border-inline-start"
+          ]
+        }],
+        "no-invalid-position-at-import-rule": null,
+        "no-descending-specificity": null,
+        "selector-class-pattern": null,
+        "scss/dollar-variable-pattern": null,
+        "selector-id-pattern": null,
+        "scss/no-global-function-names": null,
+        "selector-type-no-unknown": null,
+        "property-no-vendor-prefix": [true,
+          {
+            "ignoreProperties": ["backdrop-filter", "user-select", "initial-letter", "text-decoration", "text-stroke", "text-fill-color", "line-clamp", "box-decoration-break", "stretch"]
+          }
+        ],
+        "alpha-value-notation": "number",
+        "scale-unlimited/declaration-strict-value": [
```

Comment:
Does this rule add much value? It seems like it overlaps with the one that disallows raw hex colors and makes you use a color value instead. Are there cases that this rule catches that the raw hex rule doesn't?

Reply:
The `color-no-hex` just disallows HEX values. So, a value like `red` will pass even though there could be a red variable.
`scale-unlimited/declaration-strict-value` can be configured to catch RGB colours as well.

Reply:
Ah, yes. Good point.